obspy/io/nordic/core.py

```diff
@@ -457,7 +459,13 @@ def _read_uncertainty(tagged_lines, event):
     except ValueError:
         pass
     if errors['x_err']:
-        warnings.warn("Hypocentral Error estimates found, but not converted.")
```

Comment:
I assume you want me to put the values y_err, x_err and z_err from Nordic's "E" line converted to the units of latitude (deg), longitude (deg) and depth (m) ?  obspy's docs don't say what the units are and I don't find any equivalent fields in QuakeML.  I added the following lines after if errors['x_err']:
```
        orig.latitude_errors =  QuantityError(y_err * _km_to_deg_lat)
        orig.longitude_errors = QuantityError(x_err * _km_to_deg_lon)
        orig.depth_errors =     QuantityError(z_err * 1000.)

```
and I wrote short ``_km_to_deg_lat`` and _``km_to_deg_lon`` functions in nordic/utili.py

Reply:
This one isn't very clear in general, and different bodies do it differently.  In general for units with uncertainty though, I'm of the opinion that the uncertainty should be in the same units as the unit (which is also what nonlinloc reading does in Obspy), so I think you are right - you might be better off using something based on [locations2degrees](https://docs.obspy.org/master/packages/autogen/obspy.geodetics.base.locations2degrees.html#obspy.geodetics.base.locations2degrees) to minimise duplicate code? You may already be doing this, if so, carry on!

Reply:
Its based on [kilometers2degrees](http://docs.obspy.org/packages/autogen/obspy.geodetics.base.kilometer2degrees.html#obspy.geodetics.base.kilometer2degrees), which is easier because I can directly enter kilometers.